crates/sui-graphql-rpc/src/types/name_service.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use async_graphql::*;
+
+pub(crate) struct NameService;
+pub(crate) struct NameServiceConnection;
+
+#[allow(unreachable_code)]
+#[allow(unused_variables)]
+#[Object]
+impl NameService {
```

Comment:
In the schema `NameService` is just a `String`, i.e. it has no `id`, is this change intentional?

Reply:
see comment on placeholder to appease graphql macro

Reply:
true, will remove